config/import-control-test.xml

```diff
@@ -17,7 +17,9 @@
   <!-- Disallow hamcrest and only allow Truth -->
   <disallow pkg="org.hamcrest.CoreMatchers"/>
   <disallow pkg="org.hamcrest.MatcherAssert"/>
-  <!-- until https://github.com/checkstyle/checkstyle/issues/9142 -->
+  <!-- Disallow most assertions for Truth -->
+  <allow class="org.junit.jupiter.api.Assertions.assertThrows"/>
```

Comment:
My English is not good to read it properly.
What about
`Disallow Junit assert in favor of Truth asserts in few exceptions`

Reply:
Done.